CHANGELOG.md

```diff
@@ -50,7 +50,7 @@ Initial release.
 
 ## x.x.x
 
-This is a template. When cutting a new release, rename "master" to the release number and create a
+This is a template. When cutting a new release, rename "stable" to the release number and create a
 new, empty "Master" section.
```

Comment:
Should you also change https://github.com/material-foundation/material-text-accessibility-ios/blob/318d5100f2976e59c94643e5dcab69e7a830ee43/.travis.yml#L7?
Either s/master/stable or s/master/develop I guess.